source/MaterialXContrib/MaterialXMaya/README.md

```diff
@@ -92,21 +100,49 @@ For example:
 reloadMaterialXNode Jade_SR_jade
 ```
 
-Reloads the document and refreshes the viewport shader mapped to the specified MaterialX node. This command is useful when the contents of the document file have changed on disk, e.g. due to editing in an external XML editor.
+Reloads the document and refreshes the viewport shader mapped to the specified MaterialX node.
+This command is useful when the contents of the document file have changed on disk, e.g. due to editing in an external XML editor.
+
+### `reloadMaterialXLibraries`
+
+This command has no arguments.
+When executed, it reloads all MaterialX libraries used by the plug-in (see below for details).
 
 ## MaterialX Libraries
 
-The plug-in loads a set of standard MaterialX libraries (installed with the plug-in) at load time. When a document is loaded for a MaterialX node, it imports these libraries from memory instead of reading them from the file system.
+MaterialX documents can reference elements defined in other documents called libraries.
+When the `MaterialXMaya` plug-in is loaded by Maya, it loads a set of standard libraries
+(installed with the plug-in) and optionally a set of user libraries which can be configured via Maya option variables.
+
+Option variable `materialXLibrarySearchPaths` stores an array of file paths that are used to search for user libraries.
+
+Option variable `materialXLibraryNames` stores an array of directory names that the plug-in searches for under the paths defined by `materialXLibrarySearchPaths`.
+The plug-in loads all `.mtlx` document files found in these directories as MaterialX libraries.
 
-Users can configure additional library search paths and library directory names with the option variables `materialXLibrarySearchPaths` and `materialXLibraryNames` respectively which both hold arrays of strings. 
-The following are a couple of examples in MEL:
+Here's an example of how the two option variables can be configured in MEL:
 
 ```MEL
-optionVar -stringValueAppend materialXLibrarySearchPaths "D:/MaterialX/Teapot_Demo_MatX"
+optionVar -stringValueAppend materialXLibrarySearchPaths "D:/MaterialX/Demo/Libraries"
 optionVar -stringValueAppend materialXLibraryNames "stdlib"
 ```
 
-## Editor integration
+Please note that library and per-node document loading are independent of each other. When a MaterialX document is loaded for a `MaterialXSurfaceNode` or `MaterialXTextureNode`, it imports elements from the libraries currently in memory which speeds up document loading.
```

Comment:
About reloading.

Reply:
No need to be so polite and say 'please' in the docs. :) 
How about:
The loading of libraries and the loading of documents for a MaterialXSurfaceNode or MaterialXTextureNode are independent of each other. In the latter case, the plug-in imports elements from the libraries currently in memory, which speeds up document loading.

Reply:
Agreed, I'll make the change.